cmd/bots/internal/common/templates/alerts/base_target_alert.html

```diff
@@ -1,7 +1,4 @@
 <b>Alert type:</b> Base Target
```

Comment:
Consider this:
```html
❌<b>Base Target Alert</b>
Base target is greater than the threshold value. The threshold value is {{ .Threshold}}
{{ with .BaseTargetValues }}
{{range .}}
```

Reply:
So I only removed the details, because everything below is needed for the {{range .}}
```
Node: <code>{{ .Node }}</code>
Base Target: <code>{{ .BaseTarget }}</code>
{{end}}
{{end}}
```